vortex-array/src/array/sparse/mod.rs

```diff
@@ -157,10 +164,10 @@ impl ArrayValidity for SparseArray {
             // of true, and patch values of false.
             Self::try_new_with_offset(
                 self.indices(),
-                ConstantArray::new(false, self.len()).into_array(),
+                ConstantArray::new(true, self.indices().len()).into_array(),
```

Comment:
If you look at previous version of this logic the validity values got flipped here

Reply:
wait so this has been wrong for 3 months?

Reply:
...maybe we should add a test? i can also add in a FLUP

Reply:
I can add a test